daml/Marketplace/Custody/Model.daml

```diff
@@ -113,9 +114,13 @@ template LifecycleRequest
               election : Claim Observation Date Id -> Claim Observation Date Id -> Claim Observation Date Id
               election _ _ = deserialize choice -- will throw if the claims don't match
           Lifecycle.Result{remaining, pending} <- Lifecycle.lifecycle spot election (deserialize description.claims) t
-          (_, postAssetVersion) <- exercise versionsCid . AssetDescription.LookupOrInsert $ serialize remaining
-          payoutCids <- forA pending \(qty, asset) -> do
-            payoutDepositCid :: _ <- exercise safekeepingDepositCid $ AssetDeposit_Split [qty * assetDeposit.asset.quantity]
+          (_, postAssetVersion, _) <- exercise versionsCid AssetDescription.LookupOrInsert with newClaims = serialize remaining, assetDescription = description
+          let payoutSplitQuantities = (\(qty, _) -> qty * assetDeposit.asset.quantity) <$> pending
+          -- Next step could be:
+          -- create SettlementInstruction with buyer = customer; seller = issuer; ..
+          splitResultDeposits <- exercise safekeepingDepositCid AssetDeposit_Split with quantities = payoutSplitQuantities
```

Comment:
```suggestion
          splitResultDeposits <- exercise safekeepingDepositCid $ AssetDeposit_Split payoutSplitQuantities
```

Reply:
If you insist on changing that, I can do it but I deliberately changed it, in my opinion it is more readable (despite being a bit longer but being more explicit and DAML-ish)